packages/dnb-eufemia/src/components/date-picker/DatePickerProvider.js

```diff
@@ -304,12 +304,15 @@ export default class DatePickerProvider extends React.PureComponent {
         end_date: endDateIsValid ? format(endDate, returnFormat) : null,
         is_valid_start_date: startDateIsValid,
         is_valid_end_date: endDateIsValid,
+        partial_start_date: partialStartDate,
```

Comment:
```suggestion
        partialStartDate,
```
We can just leave it in camelCase, since we are planning to change all snake casing to camelCase at one point 🙏